csunplugged/templates/topics/programming_exercise_language_solution.html

```diff
@@ -1,20 +1,20 @@
 {% extends "topics/programming_exercise.html" %}
 
+{% load render_html_field %}
 {% load django_bootstrap_breadcrumbs %}
 
 {% block breadcrumbs %}
   {{ block.super }}
   {% breadcrumb "Solution" "" %}
 {% endblock breadcrumbs %}
+
 {% block content %}
```

Comment:
Didn't fix as it wasn't the purpose of the issue (want to go through all templates again soon and fix that plus check everything inherits properly. I'll log it as an actual issue now.